cms/djangoapps/contentstore/views/component.py

```diff
@@ -268,9 +269,11 @@ def create_support_legend_dict():
     # by the components in the order listed in COMPONENT_TYPES.
     component_types = COMPONENT_TYPES[:]
 
-    # Libraries do not support discussions
+    # Libraries do not support discussions and openassessment
+    component_not_supported_by_library = ['discussion', 'openassessment']
```

Comment:
Why don't libraries support `openassesment`? They do before this PR.
Instead of modifying the tests, you'll need to edit the code below and revert these changes:
https://github.com/edx/edx-platform/blob/8546fa6b6f669b99316cfb85878f078bc984d82d/cms/djangoapps/contentstore/views/item.py#L717 
To
```
        if category not in ['html', 'problem', 'video', 'openassessment']:
```

Reply:
I read this in the documentation here https://edx.readthedocs.io/projects/open-edx-building-and-running-a-course/en/latest/course_components/libraries.html#id13

![image](https://user-images.githubusercontent.com/7670449/93935397-2cd44880-fd42-11ea-94f5-50adbae75aee.png)

is it someother library that I misunderstood? 😅  This documentation mislead me I guess.

Reply:
Also, I found this test https://github.com/edx/edx-platform/blob/2a4c935418b552070a265e759c08f0720f9291a6/cms/djangoapps/contentstore/views/tests/test_library.py#L303 which checks that ADVANCE_PROBLEM_TYPE are not present in the Library. 


@giovannicimolin  Can you please address these comments I am feeling confused about it :(

Reply:
Oh, you're right. ORA isn't supported in libraries. Sorry for the misleading comment.
You can keep this code as is.

Reply:
No worries :)